source/components/organisms/SummaryList/SummaryList.stories.js

```diff
@@ -86,6 +86,21 @@ const SummaryStory = () => {
         answers={state}
         showSum
       />
+      <SummaryList
+        heading="Blå, ingen summa"
+        items={items}
+        categories={categories}
+        color="blue"
+        onChange={(answer, id) => {
+          setState(s => {
+            s[id] = answer;
+            return { ...s };
```

Comment:
What do s stand for?

Reply:
State, but this is bad naming, I updated it to make it a bit clearer.